lib/omniauth/strategies/saml.rb

```diff
@@ -27,6 +27,7 @@ def self.inherited(subclass)
         first_name: ["first_name", "firstname", "firstName"],
         last_name: ["last_name", "lastname", "lastName"]
       }
+      option :relay_state
```

Comment:
I think `:default_relay_state` could be a better name. It may make sense to allow it to be a proc that takes the request as a parameter as well, though I'm not sure exactly how it would be used.